### PR TITLE
changes to template for revised "check repo" instructions, including …

### PR DESCRIPTION
**Pull request by dwicinas**
…variations for Debian and Ubuntu

Changed base template file and base template file for Debian and Ubuntu

### CODE REVIEW

**Comment by dwicinas** [comment]
@drothery-edb Take a look at this when you get a chance. I think it's taking shape. I'm not completely happy with the layout of the text for the new information, but I spent a long time trying to get a better look with no luck. I'm still tweaking PEM to ensure that my changes fits smoothly into the existing information. I may not finish that today.

**Comment by dwicinas** [comment]
@drothery-edb I think this PR is looking good now. I reviewed changes for all products and they seem to be in order. One caveat is that I think DJ suggested that the command to check on the existence of a repo for SLES is not working. I'm waiting to hear back from him. @nidhibhammar, I made some changes to PEM server to fit in this new verbiage about the repo. I think everything looks good, but you might want to check PEM server to be sure.

File: product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_12.mdx [hunk 1]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_12.mdx:25]: In SLES `dnf` command doesn't work. It is `zypper`

File: product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_15.mdx [hunk 2]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_15.mdx:25]: Same comment as above

File: product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_15.mdx [hunk 3]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
...
-  !!!
+  1. Go to [EDB repositories](https://www.enterprisedb.com/repos-downloads).
+
+  1. Select the button that provides access to the EDB repo. Depending on your subscription level, you may be able to access two generations of EDB repos. EDB Repos 2.0 provides the most recent instructions.
+
+  1. Select the platform and software that you want to download.
+
+  Alternatively, you can set up the PostgreSQL community repository instead of the EDB repository. To set up the EDB PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
 
 - Install the Postgres server. See [Installing EDB Postgres Advanced Server on Linux](/epas/latest/installing/) or [Installing PostgreSQL](/supported-open-source/postgresql/installer/).
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_ppc64le/pem_sles_15.mdx:38]: In CentOS7 command is `yum` not `dnf`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx [hunk 4]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:18]: In CentOS7 command is `yum` not `dnf`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx [hunk 5]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
...
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:21]: In CentOS7 command is `yum` not `dnf`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx [hunk 6]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
...
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:23]: Need to remove extra `EDB` from the template

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_debian_10.mdx [hunk 7]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_debian_10.mdx:23]: same comment as above

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_debian_11.mdx [hunk 8]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_debian_11.mdx:23]: same comment as above

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_other_linux_8.mdx [hunk 9]
```diff
@@ -17,14 +17,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_other_linux_8.mdx:24]: Same comment as above

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_rhel_8.mdx [hunk 10]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_rhel_8.mdx:23]: Same comment as above

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_12.mdx [hunk 11]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_12.mdx:25]: Command is `zypper` not `dnf`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_12.mdx [hunk 12]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
...
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_12.mdx:27]: Remove extra `EDB` from template

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_15.mdx [hunk 13]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `dnf repolist | grep enterprisedb`
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_15.mdx:25]: Command `zypper` not `dnf`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_15.mdx [hunk 14]
```diff
@@ -20,14 +20,19 @@ Before you begin the installation process:
...
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_sles_15.mdx:27]: Remove extra `EDB`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_ubuntu_18.mdx [hunk 15]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_ubuntu_18.mdx:23]: Remove extra `EDB`

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_ubuntu_20.mdx [hunk 16]
```diff
@@ -16,14 +16,19 @@ Before you begin the installation process:
 - Set up the repository
 
   Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
+  To determine if your repository exists, enter this command:
 
-  - To set up the EDB repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
+  `apt-cache search enterprisedb`
 
-  - To set up the PostgreSQL community repository, go to [yum](https://www.postgresql.org/download/linux/redhat/) or [apt](https://wiki.postgresql.org/wiki/Apt).
+  To set up the EDB EDB repository:
```
Comment (nidhibhammar) [inline product_docs/docs/pem/9/installing/linux_x86_64/pem_ubuntu_20.mdx:23]: Remove extra `EDB`

**Comment by nidhibhammar** [comment]
@dwicinas, I think we need to change base template file for SLES and Centos 7 as well

**Comment by dwicinas** [comment]
@drothery-edb Nidhi raised an issue about the command for CentOS. Now I'm waiting to hear back from DJ as to whether the check repo command for CentOS is okay. So this PR is not quite complete.

**Comment by dwicinas** [comment]
@nidhibhammar, I fixed the duplicate "EDB EDB". I also added the correct command for SLES throughout. DJ tells me that the dnf command will work for CentOS 7 so for the sake of simplicity I'm not changing anything for CentOS 7. (BTW, DJ says that while the package manager command for CentOS 7 is yum, CentOS 7 also supports dnf.)

**Comment by dwicinas** [comment]
@drothery-edb I think this PR is finally ready for review and hopefully a merge.

**Review by nidhibhammar — DISMISSED** [review]

File: product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx (hunk 5 above)
Comment (djw-m) [reply product_docs/docs/pem/9/installing/linux_x86_64/pem_centos_7.mdx:21]: dnf ships with Centos7, it just isn't default. Yum is also present. Centos8 sees Dnf as default (and for all, going forward).

**Review by djw-m — COMMENTED** [review]
I'm noticing you are telling them to run the command but not how to interpret the output.

Generally "If no output is generated, the repository isn't installed. Follow the instructions to install the repository".

**Comment by dwicinas** [comment]
@drothery-edb Made the changes you requested

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**